src/ui/page/dataverse/dataset/dataset.tsx

```diff
@@ -9,31 +9,37 @@ import PageTemplate from '@/ui/view/dataverse/component/pageTemplate/pageTemplat
 
 const datasetMetadata: ItemGeneralMetadata[] = [
   {
+    category: 'generalMetadata',
     iconName: 'file',
     property: 'format',
     value: 'text_csv'
   },
   {
+    category: 'generalMetadata',
     iconName: 'folder',
     property: 'topic',
     value: 'industryMobilityAndEngineering'
   },
   {
+    category: 'generalMetadata',
     iconName: 'shield',
     property: 'license',
     value: 'licence:LO-FR-2_0'
   },
   {
+    category: 'generalMetadata',
     iconName: 'earth',
     property: 'geographicalCoverage',
     value: 'France'
   },
   {
+    category: 'generalMetadata',
     iconName: 'calendar',
-    property: 'temporalCoverage',
-    value: '2022-01-01T00:00:00+00:00'
+    property: 'period',
+    value: ['2022-12-31T00:00:00+00:00', '2022-01-01T00:00:00+00:00']
```

Comment:
I'm not sure I understood, I may be wrong!
But here, this is the same exemple than the example n° 4 where you have the start date which is happening after the end date, right? 
Shouldn't it display `-` instead of the from/to date?

Reply:
You are right! Thanks a lot, the bug is [in the component](https://github.com/okp4/dataverse-portal/pull/175/files#diff-4d946423f5a1c19c7fe1e8c1410d4d5e72aec0c4e9b9f5ad4c71a886dc0bab1fR46).
The properties have been reversed.